kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/marshalling/xstream/XStreamMarshaller.java

```diff
@@ -81,10 +83,13 @@
 
 public class XStreamMarshaller implements Marshaller {
 
+    public static final String XSTREAM_IGNORE_UNKNOWN = "ignoreUnknown";
```

Comment:
change the name for 

type-validation: ignore-unkown-elements

Reply:
done